### PR TITLE
Blog post about Team Time

### PR DESCRIPTION
**Pull request by RichardHarperInfinityWorks**
Please merge

### CODE REVIEW

File: _posts/2019-03-22-team-time.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+author: richard_harper
+---
+
+![Team Time!]({{ "/images/direction.png" }})
+
+# Avoiding the overrunning daily Stand-Up
+
+At Infinity Works we use an agile way of working which usually includes a daily Stand-Up. This 5-10 minute timeboxed meeting allows the team to provide an update and more importantly raise any issues they are facing. Our team uses the format of what they did yesterday (to achieve the team’s goal), what they are working on that day (to achieve the team’s goal), and anything that might prevent them from progressing with their work.
+
+Like most teams, it wasn’t uncommon for us to get side tracked or to start solutioning during a Stand-Up leading to long Stand-Up times and using up team member’s time who might have better things to be doing.
```
Comment (maurogestoso) [inline _posts/2019-03-22-team-time.md]: I don't think "solutioning" is grammatically correct (https://english.stackexchange.com/questions/67242/is-solutioning-a-correct-word/67250). You could maybe say "solving the problem"/"discussing possible solutions"?

File: _posts/2019-03-22-team-time.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Also finding time where the whole team is free or not buried in their work to discuss something could be tricky. This caused discussions to take place without all of the team and encouraged silo working. Or even worse, discouraged team members from discussing something and becoming the single point of failure.
+
+# Arriving at the solution
+
+We decided to block out some time, 30-60 minutes daily, after a Stand-Up for the team to discuss ideas/solutions, draft weeknotes, or new technical approaches they had found to improve our work. We fondly named this Team Time.
```
Comment (maurogestoso) [inline _posts/2019-03-22-team-time.md]: "or new technical approaches they had found"
During this whole sentence you use we/our but suddenly change to "they", which reads weird.

File: _posts/2019-03-22-team-time.md (hunk 1 above)
Comment (RichardHarperInfinityWorks) [reply _posts/2019-03-22-team-time.md]: Should be resolved please re-review

File: _posts/2019-03-22-team-time.md (hunk 2 above)
Comment (RichardHarperInfinityWorks) [reply _posts/2019-03-22-team-time.md]: Should be resolved please re-review

**Comment by maurogestoso** [comment]
Looks good to me!

**Pull request merged**